mindsdb/integrations/handlers/langchain_handler/langchain_handler.py

```diff
@@ -291,8 +304,13 @@ def run_agent(self, df: pd.DataFrame, agent: AgentExecutor, args: Dict, pred_arg
         def _invoke_agent_executor_with_prompt(agent_executor, prompt):
             if not prompt:
                 return ''
-
-            answer = agent_executor.invoke(prompt)
+            try:
+                answer = agent_executor.invoke(prompt)
+            except Exception as e:
+                answer = str(e)
```

Comment:
Seeing agent errors which raise an unnecessary exception, this is a simple way to handle to catch and only raise when absolutelt necessary

Reply:
I found without this I would still get errors (even with exception parser)